modules/translation/translation.go

```diff
@@ -57,8 +59,13 @@ func InitLocales() {
 	matcher = language.NewMatcher(tags)
 	for i := range setting.Names {
 		key := "locale_" + setting.Langs[i] + ".ini"
-		if err := i18n.SetMessageWithDesc(setting.Langs[i], setting.Names[i], localFiles[key]); err != nil {
-			log.Fatal("Failed to set messages to %s: %v", setting.Langs[i], err)
+		if err = i18n.SetMessageWithDesc(setting.Langs[i], setting.Names[i], localFiles[key]); err != nil {
+			if errors.Is(err, i18n.ErrLangAlreadyExist) {
+				// just log if lang is already loaded since we can not reload it
+				log.Warn("Can not load language '%s' since already loaded", setting.Langs[i])
+			} else {
+				log.Fatal("Failed to set messages to %s: %v", setting.Langs[i], err)
```

Comment:
Make this:

```suggestion
				log.Error("Failed to set messages to %s: %v", setting.Langs[i], err)
```

Killing a server and causing a docker loop for a problem with a possibly unused or not found locale file is just nasty.